### DISCUSSION TITLE
Why does documentation include .venv note?

### DISCUSSION THREAD

**Discussion by pamelafox** [Azure/azure-dev#1071] [Q&A] [answered]
Referring to this tutorial on azd-ification:
https://learn.microsoft.com/en-us/azure/developer/azure-developer-cli/make-azd-compatible?pivots=azd-convert

The pipeline section ends with this note:

`.venv` should be added to the `.gitignore` file

I've been scratching my head trying to figure out why it's in that section, as it doesn't feel related. The tutorial never mentions venv, and presumably, if the developer has decided to use venv independently, they figured out to add to their gitignore earlier. Could you explain the reasoning? Why is it so necessary at that step?

Thanks!

CC @puicchan

**Comment by puicchan** [comment, marked as answer]
For in case the customer did this optional step: 

![image](https://user-images.githubusercontent.com/8701892/202600944-d1c7e6cc-448e-4304-b22e-9b0a064e6014.png)
